code/game/objects/structures/transit_tubes/station.dm

```diff
@@ -147,7 +147,7 @@
 		sleep(OPEN_DURATION + 2)
 		pod_moving = 0
 		if(!QDELETED(pod))
-			pod.air_contents.share(loc.return_air()) //mix the pod's gas mixture with the tile it's on
+			pod.air_contents.share(loc.return_air(), 1) //mix the pod's gas mixture with the tile it's on
 
```

Comment:
You don't need to achieve the turf, just the pod. achieve the pod's air_contents before calling share() and that should fix this issue.